### ISSUE TITLE
Enhanced Accuracy for Ticket Categorization

### ISSUE THREAD

**Issue by jyotirmayapatro** [jyotirmayapatro/blank-app#16]
Fine-tune pre-trained models with additional labeled data from the client’s historical ticket database.

Experiment with different models to compare performance.

**Comment by jyotirmayapatro** [comment]
We have improved accuracy while implementing Open AI and Gemini model.
